clusterdock/topologies/cdh/actions.py

```diff
@@ -38,6 +39,21 @@
 logger.setLevel(logging.INFO)
 
 DEFAULT_CLOUDERA_NAMESPACE = Constants.DEFAULT.cloudera_namespace # pylint: disable=no-member
+KDC_HOST_NAME = 'kdc'
+KDC_KRB5_CONF_FILENAME = '/etc/krb5.conf'
+KDC_CONF_FILENAME = '/var/kerberos/krb5kdc/kdc.conf'
+KDC_ACL_FILENAME = '/var/kerberos/krb5kdc/kadm5.acl'
+CLDR_MGR_PRINCIPAL_USER = 'cloudera-scm/admin'
+CLDR_MGR_PRINCIPAL_PW = 'cldadmin'
+KDC_KEYTAB_FILENAME = '/tmp/kerberos/sdc.keytab'
```

Comment:
Should we perhaps pick a location not in `/tmp`. Worried that processes may come in and clear out `/tmp` regularly for long-running clusterdock clusters.